fea-rs/src/compile/lookups.rs

```diff
@@ -838,6 +842,58 @@ impl SomeLookup {
         }
     }
 
+    fn flags(&self) -> LookupFlag {
```

Comment:
This is gross and not your fault, I should really have a macro or something to handle these enums. In any case maybe we can improve it somewhat by returning `LookupFlagInfo`, so we don't need two separate branchy methods?